filebrowser/main.tf

```diff
@@ -14,6 +14,24 @@ variable "agent_id" {
   description = "The ID of a Coder agent."
 }
 
+variable "workspace_name" {
+  type = string
+  default = ""
+  description = "Set this and owner_name to serve filebrowser from subdirectory."
+}
+
+variable "owner_name" {
+  type = string
+  default = ""
+  description = "Set this and workspace_name to serve filebrowser from subdirectory."
+}
+
```

Comment:
And this from https://registry.terraform.io/providers/coder/coder/latest/docs/data-sources/workspace_owner
```suggestion
```